amm/traits/stable_pool.rs

```diff
@@ -27,46 +27,64 @@ pub trait StablePool {
     #[ink(message)]
     fn fee_receiver(&self) -> Option<AccountId>;
 
-    /// Updates cached token rates if expired and
-    /// returns current tokens rates with precision of 12 decimal places.
+    /// Updates cached token rates.
```

Comment:
It doesn't always update, right? Only if the previous update has been done in a block which is lower than the current one.
```suggestion
    /// Updates cached token rates if there was a new block since the previous update.
```